index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My-Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="menu-bar">
+            <div class="hamburger-container">
+                <img src="Icons/Icon-Menu.png" alt="menu bar">
+            </div>
+        </nav>
+    </header>
+    <section class="header-container">
+      <div>
+        <h1 class="bio">
+            Hey there. <br> I'm Yakub. <br>
+            <span>I’m a software developer</span> 
+        </h1>
+        <p class="intro">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </p>
+      </div>
+        <div class="social-media">
+            <ul>
+            <li>
+                <a href="#">
+                    <img src="Icons/twitter.svg" alt="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="Icons/Linkedin-icon.svg" alt="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="Icons/medium.svg" alt="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="Icons/github.svg" alt="twitter">
+                </a>
+            </li>
+            <li>
+                <a href="#">
+                    <img src="Icons/angellist.svg" alt="twitter">
+                </a>
+            </li>
+        </ul>
```

Comment:
- [ ] Kindly fix the social icons they are showing
<img width="504" alt="Screenshot 2023-03-24 at 06 26 25" src="https://user-images.githubusercontent.com/81444338/227536690-253fccf4-85d7-44c1-a9b0-a292b9db29f9.png">

Reply:
@Kossi-stack 
Kindly recheck the icons from your side because all the social icons are showing from here.
I have also reached out to another reviewer and he confirms all the icons are showing
![Screenshot from 2023-03-24 15-47-22](https://user-images.githubusercontent.com/89491845/227558426-82eedb04-2674-4756-b6dc-c90c5dd1901e.png)